lib/nerves_hub/config.ex

```diff
@@ -25,4 +25,17 @@ defmodule NervesHub.Config do
            {:url_port, "URL_PORT", default: "443", map: &String.to_integer/1},
            {:url_scheme, "URL_SCHEME", default: "https"}
          ])
+
+  config :audit_logs,
+         env([
+           {:enabled, "TRUNATE_AUDIT_LOGS_ENABLED", default: "true", map: &to_boolean/1},
```

Comment:
```suggestion
           {:enabled, "TRUNATE_AUDIT_LOGS_ENABLED", default: "false", map: &to_boolean/1},
```
I think this was previously defaulted to `false` to allow users to decide if when to remove data instead